cmake/ffig.cmake

```diff
@@ -55,11 +55,25 @@ function(ffig_add_library)
     set(ffig_invocation "${ffig_invocation};_cpp.h.tmpl")
     set(ffig_outputs "${ffig_outputs};${ffig_output_dir}/${module}_cpp.h")
   endif()
+  if(ffig_add_library_GO)
+    set(ffig_invocation "${ffig_invocation};go.tmpl")
+    set(ffig_outputs "${ffig_outputs};${ffig_output_dir}/src/${module}/${module}.go")
+  endif()
+
+  if(ffig_add_library_GO)
```

Comment:
Tolerable, but not ideal. As the number of optional binding languages grows, we will need a better mechanism to support custom steps in the generation.